index.html

```diff
@@ -12,10 +12,21 @@
       href="https://fonts.googleapis.com/css2?family=Architects+Daughter&display=swap"
       rel="stylesheet"
     />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700&family=Roboto+Mono&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inconsolata&display=swap"
+      rel="stylesheet"
+    />
     <h2>Greetings retard</h2>
   </head>
 
   <body>
+    <script src="js/bugfix.js"></script>
```

Comment:
haha

Reply:
? please take this more seriously

Best Regards
Team Natron

Reply:
Why did you not use correct indentation @0us  ? ? ?